src/pages/index.js

```diff
@@ -118,40 +253,59 @@ function getCardElement(data) {
 
   const cardImage = cardElement.querySelector(".card__image");
 
-  const cardLikeBtn = cardElement.querySelector(".card__like-button");
+  const cardLikeButton = cardElement.querySelector(".card__like-button");
 
-  const cardDeleteBtn = cardElement.querySelector(".card__delete-button");
+  const cardDeleteButton = cardElement.querySelector(".card__delete-button");
 
   cardImage.src = data.link;
 
   cardNameEl.textContent = data.name;
 
   cardImage.alt = data.alt;
 
-  cardLikeBtn.addEventListener("click", () => {
-    cardLikeBtn.classList.toggle("card__like-button_liked");
+  cardDeleteButton.addEventListener("click", (evt) => {
+    const cardElement = evt.target.closest(".card");
+    selectedCardId = data._id;
+    selectedCardElement = cardElement;
+    openModal(deleteModal);
   });
 
```

Comment:
* When I reload the page, all **red** hearts become **transparent** again.

You should check `isLiked` here

```js


  if (data.isLiked) {
    cardLikeButton.classList.add("card__like-button_liked");
  }
```